linux_os/guide/system/auditing/auditd_configure_rules/file_ownership_var_log_audit_stig/oval/shared.xml

```diff
@@ -6,7 +6,16 @@
         <extend_definition comment="log_file not set in auditd.conf" definition_ref="auditd_conf_log_file_not_set" negate="true" />
         <criterion comment="audit log files are root owned" test_ref="test_user_ownership_audit_log_files" />
       </criteria>
+      {{% if product in ["ol8", "rhel8"] %}}
```

Comment:
this will affect Ubuntu STIG, as I added those bits.
You probably want to do an if specific to your product instead.